mne/chpi.py

```diff
@@ -402,7 +403,11 @@ def _setup_hpi_struct(info, model_n_window,
     else:  # == 'multipole'
         coils = _prep_mf_coils(info)
     scale = make_ad_hoc_cov(info, verbose=False)
-    scale = _get_whitener_data(info, scale, meg_picks, verbose=False)[0]
+
+    scalings = _handle_default('scalings', None)
+    # WTF is the whitener called scale ! cc @larsoner...
```

Comment:
yes because all it does is scale each channel (it's a diagonal matrix), feel free to change if you want